terra/tests.py

```diff
@@ -584,12 +584,12 @@ def test_get_fund_employee_list(self):
 
     def test_fund_report(self):
```

Comment:
@aprigge Since the acceptance criteria say this:
`Expenses are in the fiscal year if:
    Payment of the expenses was made in the designated fiscal year (actual travel dates are irrelevant
`
I think the test(s) should reflect this.  There should be sample data outside of the current FY, and a test should verify that this data is not included in the report's totals. 

Otherwise, this looks good.

Reply:
Thanks, @akohler . I'll make those changes.

Reply:
@akohler I changed a date for an actual expense for FY19 with treq dates in FY20. I think the `test_fund_report` test verifies the data is not included in the query.
-I've also changed the variable names in the Fund export view
-I changed the `test_subtotal` to use `get_individual_data_employee` function instead of the unit report's `get_individual_data`.